src/utils/helper.ts

```diff
@@ -108,6 +108,10 @@ export function isDict(v) {
     return typeof v==='object' && v!==null && !(v instanceof Array) && !(v instanceof Date);
 }
 
+export function ToJsonString(str: string): string {
+    return JSON.stringify(str).split(/[\r\n]+/).join("").split("\\").join("\\\\").split("'").join("\\'")
```

Comment:
Why do you need this? Is it for deal with the swagger example string? how about use jsos load first and then json dumps? Not sure the exact method in typescript

Reply:
just commit a correct on this. 
This function is actually doing "dumps" for any thing. The original signature is misleading.